server/db/recreate-schema.sql

```diff
@@ -5,5 +5,6 @@ DROP TABLE if exists users;
 CREATE TABLE users (
   id SERIAL PRIMARY KEY,
   email    VARCHAR(200) NOT NULL,
+  role     VARCHAR(15) NOT NULL
```

Comment:
```suggestion
  role     VARCHAR(15) NOT NULL,
```
It looks like we're missing a comma at the end of this line. It's good practise to test your changes before requesting review!

Reply:
Hi Coung, you are right. I think I can change it on my side and then git push on github? Is it ok or I need to use button "Commit suggestion"?